### ISSUE TITLE
Hiding zoom controls

### ISSUE THREAD

**Issue by trixon** [sothawo/mapjfx#59]
Is there a good and easy way to hide the zoom controls in the upper left corner of the map?
I'm able to do it by replacing 
```
controls: ol.control.defaults({
    zoom: interactive
}),
```

with 
`controls: [],`
in mapview.js.

However, placing this altered mapview.js in my project does not override it, this could be due to NetBeans class loaders, I don't know.

The reason for wanting this is that I already have a zoom control, the one in the lower left corner of:
![alt tag](https://mapton.org/wp-content/uploads/2019/08/2019-08-19-085507_1459x892_scrot.png)

**Comment by sothawo** [comment]
Hi Patrick,
using your own _mapview.js_ will not work, as this is loaded from the same path where the html is loaded from; and placing that in a custom folder where it is found by the classloader first would mean that you need to put *all* the html and js files there - including OpenLayers. I would not guarantee that this is working.

My idea would be to change the initialization of the MapView Java object (and of the underlaying Javaascript object). At the moment there are the two parameters for the projection and the interactive flag; This is messy and I would like to encapsulate this into a separate configuration object which is then internally transferred into an configuration object for the Javascript class. This would leave room for extending and adding configuration options.

I'll put this on my list, but it might be the next weekend before I find time to implement this.

btw, although I know that some people are using mapjfx, this is the first screenshot I see of an application that integrates it, looks nice

**Comment by sothawo** [comment]
implemented in versions 1.27.0 and 2.8.0

tl;dr
configure your `MapView` instance with:
```
        mapView.initialize(Configuration.builder()
            .showZoomControls(false)
            .build());
```

**Comment by trixon** [comment]
Works great, thanks!
